maistra/bazelrc

```diff
@@ -0,0 +1,5 @@
+# FIXME: https://issues.redhat.com/browse/OSSM-1201
+build --cxxopt -w
```

Comment:
Do we still need this? (No objections to merging as-is and mopping up later if so, just curious)

Reply:
We do. I had a hard time trying to workaround some warnings/errors (-Werror=xxx) in the proxy build. Adding back this flag (we had it in 2.1) is the way to move on with it.